include/script_api/common.h

```diff
@@ -162,7 +162,7 @@ ApiStatus SwitchMessage(ScriptInstance* script, s32 isInitialCall);
 ApiStatus ShowChoice(ScriptInstance* script, s32 isInitialCall);
 ApiStatus CloseChoice(ScriptInstance* script, s32 isInitialCall);
 ApiStatus CancelMessage(ScriptInstance* script, s32 isInitialCall);
-ApiStatus CancelMessageAndBlock(ScriptInstance* script, s32 isInitialCall);
+ApiStatus CancelMessageAndBlock(); // ???
```

Comment:
?

Reply:
It seems to not be a script function based on another function that calls it, unless we have stuff very wrong. I'll investigate but..yeah

Reply:
we should be able to just look to see if this is either used or returns 2